express/parsers/settings.py

```diff
@@ -18,10 +18,10 @@ class Constant(object):
     _Nav = 6.0221367e23  # Avogadro number
     _k = 1.380658e-23  # Boltzmann constant, J/K
     _amu = 1.6605402e-27  # atomic mass unit, kg
-    BOHR = 4e10 * pi * _eps0 * _hbar ** 2 / _me / _e ** 2  # Bohr radius
+    BOHR = 4e10 * pi * _eps0 * _hbar ** 2 / _me / _e ** 2  # Bohr radius in angstrom
     eV = 1.0
-    HARTREE = _me * _e ** 3 / 16 / pi ** 2 / _eps0 ** 2 / _hbar ** 2
-    RYDBERG = 0.5 * HARTREE
+    HARTREE = _me * _e ** 3 / 16 / pi ** 2 / _eps0 ** 2 / _hbar ** 2  # in eV
+    RYDBERG = 0.5 * HARTREE  # in eV
```

Comment:
Nit: this could be moved to and reused from the `periodic-table`, I guess - we defined it there recently too, right?

Reply:
`periodic-table` has a (small) conversion table yes, but it is written in JS. I think it may be easier to use the collection of constants in `scipy` instead: https://docs.scipy.org/doc/scipy/reference/constants.html

Reply:
Good idea - let' reuse from a scipy, a well-established place

Reply:
Let's put a task to deal with it